src/store/mutations.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash'
+
+export default {
+  UPDATE_LIST (state, payload) {
+    const id = { 'id': payload.id }
+    if (payload.list) {
+      const movie = _.find(state.list, id)
+      movie.list = false
+      state.list = _.without(state.list, movie)
```

Comment:
see https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_without for alternative

Reply:
oops - so maybe there isn't a native equivalent for objects

Reply:
See 85e58f9